services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1703,6 +1749,19 @@ void GraphBuilderOrt::AddPool2dOperation(const mojom::Pool2d& pool2d) {
                          attributes);
 }
 
+void GraphBuilderOrt::AddPreluOperation(const mojom::Prelu& prelu) {
+  const std::string node_name = GenerateNextOperationName(prelu.label);
+  const std::string input_name = GetOperandNameById(prelu.input_operand_id);
+  const std::string slope_name = GetOperandNameById(prelu.slope_operand_id);
+  const std::string output_name = GetOperandNameById(prelu.output_operand_id);
+
+  std::array<const char*, 2> input_names = {input_name.c_str(),
+                                            slope_name.c_str()};
+  std::array<const char*, 1> output_names = {output_name.c_str()};
+
```

Comment:
ONNX Prelu requires https://onnx.ai/onnx/operators/onnx__PRelu.html

> The shape of slope can be smaller than first input X; if so, its shape must be unidirectional broadcastable to X

While WebNN allows input and slope to be bidirectionally broadcastable.

Worth adding a TODO to emulate if slope is not unidirectional broadcastable to input.

Reply:
Done in new commit. Please take another look. Thanks!